templates/csv.yaml

```diff
@@ -400,7 +400,12 @@ spec:
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
                   image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress
                   imagePullPolicy: Always
+                  ports:
+                    - containerPort: 9090
+                      name: metrics
                   env:
+                    - name: METRICS_DOMAIN
+                      value: knative.dev/serving-ingress
```

Comment:
Why not `knative.dev/serving`?

Reply:
no preference just because it is for ingress.